wcfsetup/install/files/lib/form/LostPasswordForm.class.php

```diff
@@ -27,6 +28,7 @@
 class LostPasswordForm extends AbstractCaptchaForm
 {
     const AVAILABLE_DURING_OFFLINE_MODE = true;
+    const ALLOWED_RESETS_PER_24H = 3;
```

Comment:
Maybe increase this a little:
```suggestion
    private const ALLOWED_RESETS_PER_24H = 5;
```